static/summary.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
```

Comment:
rename this file to be `metricsummary.html`

Reply:
Done.